docs/sources/api/_index.md

```diff
@@ -1237,8 +1237,9 @@ will be sent over the WebSocket multiple times.
 
 ### `GET /api/prom/query`
 
-> **WARNING**: `/api/prom/query` is DEPRECATED; use `/loki/api/v1/query_range`
-> instead.
+{{% admonition type="warning" %}}
```

Comment:
I think your script missed one (because it's inconsistent in the docs)
Up at line 1191 there's a deprecation note that should be changed to a warning.